packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -355,6 +361,183 @@ class Actions extends InheritedWidget {
   }
 }
 
+/// A widget that combines the functionality of [Actions], [Shortcuts],
+/// [MouseRegion] and a [Focus] widget to create a detector that defines actions
+/// and key bindings, and will notify that the focus or hover highlights should
```

Comment:
> ... and will notify that ...

Notify whom? :) 

Perhaps instead "and provides callbacks for handling focus and hover highlights"?

Reply:
Done